test/help/help.test.js

```diff
@@ -14,111 +11,31 @@ describe('help', () => {
 
         expect(exitCode).toBe(0);
         expect(stderr).toBeFalsy();
-
-        expect(stdout).toContain('webpack [entries...] [options]');
-        expect(stdout).toContain('webpack [command] [options]');
-        expect(stdout).toContain(helpDefaultHeader);
-        expect(stdout).toContain('Options:');
-        expect(stdout).toContain('--merge'); // minimum
-        expect(stdout).not.toContain('--cache-type'); // verbose
-
-        if (!isMacOS) {
-            expect(stdout).toContain('Global options:');
-            expect(stdout).toContain('Commands:');
-            expect(stdout.match(/build\|bundle\|b/g)).toHaveLength(1);
-            expect(stdout.match(/watch\|w/g)).toHaveLength(1);
-            expect(stdout.match(/version\|v/g)).toHaveLength(1);
-            expect(stdout.match(/help\|h/g)).toHaveLength(1);
-            expect(stdout.match(/serve\|s/g)).toHaveLength(1);
-            expect(stdout.match(/info\|i/g)).toHaveLength(1);
-            expect(stdout.match(/init\|c/g)).toHaveLength(1);
-            expect(stdout.match(/loader\|l/g)).toHaveLength(1);
-            expect(stdout.match(/plugin\|p/g)).toHaveLength(1);
-            expect(stdout.match(/migrate\|m/g)).toHaveLength(1);
-            expect(stdout.match(/configtest\|t/g)).toHaveLength(1);
-            expect(stdout).toContain("To see list of all supported commands and options run 'webpack --help=verbose'.");
-            expect(stdout).toContain('CLI documentation: https://webpack.js.org/api/cli/.');
-            expect(stdout).toContain('Made with ♥ by the webpack team.');
-        }
+        expect(stripAnsi(stdout)).toMatchSnapshot();
```

Comment:
Why not making use of the `jest-serializer-ansi` snapshot serializer?
https://github.com/webpack/webpack-cli/blob/e13082221c2da01d8b8215ebc936474bf3ca1582/package.json#L73

Reply:
yes much cleaner, lets do it